lightning/src/ln/blinded_payment_tests.rs

```diff
@@ -41,6 +41,9 @@ use lightning_invoice::RawBolt11Invoice;
 use types::features::Features;
 use crate::blinded_path::BlindedHop;
 
+#[cfg(trampoline)]
```

Comment:
General trampoline question: suppose a receiver doesn't support trampoline and also doesn't want to do blinded paths. Is it still possible to pay them via a trampoline node? Thinking of the case where an LSP is always the trampoline (offloading routing) and needs to be able to pay any type of invoice.

Reply:
It surprises me that this is not part of it anymore. I'd think that for an LSP doing the routing this is really important to have, otherwise it cannot route for every invoice?

Reply:
Discussed in spec meeting. Bastien will write a blip for it, so we can support the same protocol if we want to in the future.